src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -114,6 +114,19 @@ void GeotaggedImagesPlugin::Load(sensors::SensorPtr sensor, sdf::ElementPtr sdf)
         _maxZoom = sdf->GetElement("maximum_zoom")->Get<float>();
     }
 
+    if (sdf->HasElement("stream_uri")) {
+        _streamURI = sdf->GetElement("stream_uri")->Get<int>();
+    }
+    if (sdf->HasElement("system_id")) {
+        _systemID = sdf->GetElement("system_id")->Get<int>();
+    }
+    if (sdf->HasElement("component_id")) {
+        _componentID = sdf->GetElement("component_id")->Get<int>();
```

Comment:
I don't think this needs to be configurable, just fix it to `MAV_COMP_ID_CAMERA`

Reply:
@Jaeyoung-Lim I think it depends on the setup. For example, if a vehicle has two cameras, then the system ID would be the same but the camera ID would be different in order to be able to differentiate them.

Reply:
Right, but I think that should be enabled separately. I am more concerned that the user will be able to assign arbitrary componentIDs, which will cause problems